cpp/tests/sparse/spectral_matrix.cu

```diff
@@ -84,6 +86,62 @@ TEST(Raft, SpectralMatrices)
   EXPECT_ANY_THROW(cnstr_mm2());  // because of nullptr ptr args
 }
 
+TEST(Raft, ComputeGraphLaplacian)
```

Comment:
Can we also pull this into `sparse/linalg` namespace?

Reply:
Addressed in https://github.com/rapidsai/raft/pull/2568/commits/08e9f0ff132d0b43bc875c1c08668ed7b06480c4